### PR TITLE
[Estuary] Fix order of TVShowTitle and Title

### PR DESCRIPTION
**Pull request by DaVukovic**
## Description

The ordering at the DialogVideoInfo screen was wrong in regard of the TV show title and the episode title.

## Motivation and Context

Fix the ordering mentioned here:

https://forum.kodi.tv/showthread.php?tid=262373&pid=2844222#pid2844222

## How Has This Been Tested?

Tested on latest Kodi master

## Screenshots (if appropriate):
before:
grafik

after:
grafik

## Types of change

- [x] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by DaVukovic** [comment]
@ronie friendly reminder to take a look if you have the time. Don't worry if not.

**Comment by HitcherUK** [comment]
Looks good to me, thanks.

File: addons/skin.estuary/xml/Variables.xml [hunk 1]
```diff
@@ -258,15 +258,15 @@
 	</variable>
 	<variable name="VideoInfoMainLabelVar">
 		<value condition="String.IsEqual(ListItem.DBType,set)">$INFO[ListItem.Title]</value>
-		<value condition="!String.IsEmpty(ListItem.Title)">$INFO[ListItem.Title]$INFO[ListItem.Year, ([COLOR grey],[/COLOR])]</value>
+		<value condition="!String.IsEmpty(ListItem.TVShowTitle)">$INFO[ListItem.TVShowTitle]$INFO[ListItem.Year, ([COLOR grey],[/COLOR])]</value>
```
Comment (ronie) [inline addons/skin.estuary/xml/Variables.xml]: ListItem.Title is used for other types of content, so please do not remove that line.
just place the ListItem.TVShowTitle one above the ListItem.Title one and you should be fine.

**Review by ronie — CHANGES_REQUESTED** [review]

File: addons/skin.estuary/xml/Variables.xml (hunk 1 above)
Comment (DaVukovic) [reply addons/skin.estuary/xml/Variables.xml]: Ok, will do

**Comment by DaVukovic** [comment]
@ronie done ;)

**Comment by ronie** [comment]
thx!
can you do a backport as well?

**Pull request merged**